sagemaker-triton/nlp_bert/triton_nlp_bert.ipynb

```diff
@@ -0,0 +1,681 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    text = "Amazon SageMaker helps data scientists and developers to prepare, build, train, and deploy high-quality machine learning (ML) models quickly by bringing together a broad set of capabilities purpose-built for ML."
</pre><p>This sentence is defined twice in the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
Done